polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -2170,14 +2170,7 @@ MLIRASTConsumer::getOrCreateGlobal(const ValueDecl &VD, std::string Prefix,
 
     MLIRScanner MS(*this, module, LTInfo);
     mlir::Block B;
-    // In case of device function, we will put the block in the forefront of
```

Comment:
It is unnecessary, because at line 2101:
```
  mlir::OpBuilder Builder(module->getContext());
  if (FuncContext == FunctionContext::SYCLDevice)
    Builder.setInsertionPointToStart(getDeviceModule(*module).getBody());
  else {
    assert(FuncContext == FunctionContext::Host);
    Builder.setInsertionPointToStart(module->getBody());
  }
  ```